staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope.go

```diff
@@ -139,28 +117,16 @@ func (t *envelopeTransformer) TransformToStorage(data []byte, context value.Cont
 
 var _ value.Transformer = &envelopeTransformer{}
 
-// addTransformer inserts a new transformer to the Envelope cache of DEKs for future reads.
-func (t *envelopeTransformer) addTransformer(encKey []byte, key []byte) (value.Transformer, error) {
+// newTransformer inserts a new transformer to the Envelope cache of DEKs for future reads.
```

Comment:
is this godoc still accurate?

Reply:
No, it was not - thanks for catching this.
Update the godoc.